.claude/skills/worldai-browser-login.md

```
@@ -0,0 +1,200 @@
+    action="click",
+    selector="//button[contains(text(), 'Sign in with Google')]"
+)
+```
+
+### Step 3: Complete OAuth in Popup
+
+The Firebase OAuth popup will open. The MCP browser cannot track the popup window, but:
+- If you've previously signed in with Google in the same browser profile, it may auto-complete
+- Manual intervention may be needed to click your Google account
+- After successful auth, the main page will show your email in the header
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)